build.ps1

```diff
@@ -34,6 +34,12 @@ Exec-Block { & "$SCRIPT_ROOT\init-tools.cmd" } | Out-Host
 $CLIPATH = "$SCRIPT_ROOT\Tools\dotnetcli"
 $SDKPATH = "$CLIPATH\sdk\$SdkVersion"
 
+$BINLOG_ROOT = "$SCRIPT_ROOT\bin\obj\"
```

Comment:
I think this is probably worse then the original problem. I think I like doing the gitignore over having to initialize/create this directory directly in the scripts.

Reply:
Alright, reverting. Didn't realize this would be needed until I saw it in CI.